doc/reports/prelude.rst

```diff
@@ -0,0 +1,82 @@
+
+Zephyr Test Result Report
+############################
+
+
+OVERVIEW
+************************************

```

Comment:
this should be same length as the title.

Reply:
ok, done.